eng/pipelines/pipeline-generation.yml

```diff
@@ -68,7 +70,9 @@ jobs:
           --variablegroup ${{ parameters.Release_Secrets_for_GitHubIO_Publishing }}
           --variablegroup ${{ parameters.Secrets_for_DevOps_Notification_Configuration_and_User_Resolution }}
           --variablegroup ${{ parameters.APIReview_AutoCreate_Configurations }}
-        TestOptions: '--variablegroup ${{ parameters.Secrets_for_Resource_Provisioner }}'
+        TestOptions: >-
+          --variablegroup ${{ parameters.Secrets_for_Resource_Provisioner }}
+          --variablegroup ${{ parameters.Test_Secrets_for_JS_Live_Tests }}
```

Comment:
If we do take this change we will also need to update https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/prepare-pipelines.yml#L10